src/miner.h

```diff
@@ -135,6 +135,7 @@ class BlockAssembler
     bool fIncludeWitness;
     unsigned int nBlockMaxWeight;
     CFeeRate blockMinFeeRate;
+    std::chrono::seconds m_max_tx_time;
```

Comment:
this name is pretty ambiguous.

`m_skip_inclusion_until` or something in that vicinity?

Reply:
fixed